internal/services/network/public_ip_prefix_resource.go

```diff
@@ -63,6 +63,17 @@ func resourcePublicIpPrefix() *pluginsdk.Resource {
 				}, false),
 			},
 
+			"sku_tier": {
+				Type:     pluginsdk.TypeString,
+				Optional: true,
+				Default:  string(publicipprefixes.PublicIPPrefixSkuTierRegional),
+				ForceNew: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					string(publicipprefixes.PublicIPPrefixSkuTierGlobal),
+					string(publicipprefixes.PublicIPPrefixSkuTierRegional),
+				}, false),
```

Comment:
This can be replaced with 
```suggestion
				ValidateFunc: validation.StringInSlice(PossibleValuesForPublicIPPrefixSkuTier(), false),
```

Reply:
Lovely, thank you.